bridge-ffi/build.rs

```diff
@@ -0,0 +1,16 @@
+extern crate cbindgen;
+
+use std::env;
+
+fn main() {
+    let crate_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
+
+    cbindgen::Builder::new()
+        .with_crate(crate_dir)
+        // .with_no_includes()
```

Comment:
Leftover comment line

Reply:
I was struggling with this because I couldn't decide whether I wanted the includes. I think I am more sure I do now, so I can remove this.

Reply:
Removed line